docs/administration/service_management.md

```diff
@@ -1,132 +1,170 @@
 
-At the top right corner of the service management page, you will also find backup and restore buttons for creating and restoring backups for the services configurations.
+## Creating and Restoring Service Config Backups
 
-The backup button which looks like an "*arrow pointing down*" will create a yml with a filename of the following format: `<FQDN>_service_backup.yml`. The file will automatically be downloaded by your browser in your download directory.
+Backup and restore functionality allows you to save and recover service configurations.
 
-Once you want to restore the backup in your system, you can simply click the restore button, "*clock with a counter clockwise arrow*", This will open a modal window with an empty textbox.
+To manage backups:
+1. Click the "*arrow pointing down*" button to download the current services configuration as a YAML file, named `<FQDN>_service_backup.yml`.
+2. To restore a backup, click the "*clock with counter-clockwise arrow*" button. Paste the backup content into the textbox and click "*Restore*" to reinstate the services configurations to their saved state.
 
 ![Service restore](./images/service_restore.png)
 
-Simply paste the content of the backup created earlier in the text box and hit the "*Restore*" button to restore the services configurations to their backed-up values.
-
 ## Service Listing Overview
 
-Let's look at the columns and data in the service table:
+The service listing table provides comprehensive information on each of the services configured in Assemblyline. Below is a detailed explanation of each column and the type of data it represents:
 
 ![Service Management page](./images/service_mgmt_page.png)
 
-The "Name" column contains the service name, and the "Version" column contains the version of the service that is running on Assemblyline. 
+**Name**
+
+The **Name** column contains the name of the service. This is the identifier that you will use to refer to this service throughout the system.
+
+**Version**
+
+The **Version** column lists the current running version of the service.
+
+**Category**
+
+The **Category** column categorizes the services into predefined groups. This helps in organizing services and understanding the role each service plays. Available categories out-of-the-box include:
+
+- **Antivirus**: Services that scan files for malware.
+- **Dynamic Analysis**: Services that execute the file in a sandbox environment to observe behaviors.
+- **External**: Services that rely on external data sources or submit data to external systems.
+- **Extraction**: Services that extract files from archives or other compound file formats.
+- **Filtering**: Services that eliminate certain files from being processed further.
+- **Internet Connected**: Services that need to connect to the internet to fetch updates or check data.
+- **Networking**: Services that analyze network traffic or related data.
+- **Static Analysis**: Services that analyze files without executing them.
```

Comment:
I could not find a definition for each category, and I think it would be valuable to have one.  Those are LLM generated and are obviously not perfect.  Can you come up with a rough one for each and I can polish them using an LLM.

Reply:
Antivirus: Services that use built-in signatures to scan files for viruses and malware.
Dynamic Analysis: No change
External: No change
Extraction: No change
Filtering: No change
Internet Connected: Services that allow executed files to connect to the internet in order to receive instructions or download additional files. 
Networking: No change
Static Analysis: No change.